pages.es/linux/userdel.md

```diff
@@ -1,21 +1,17 @@
 # userdel
 
 > Elimina una cuenta de usuario o elimina un usuario de un grupo.
-> Nota: todos los comandos deben ser ejecutados como root.
+> Ver también: `users`, `useradd`, `usermod`.
 > Más información: <https://manned.org/userdel>.
 
 - Elimina un usuario:
 
-`userdel {{nombre}}`
+`sudo userdel {{nombre}}`
 
-- Elimina un usuario junto con su directorio home y mail spool:
+- Elimina un usuario en otro directorio raíz(root):
 
-`userdel --remove {{nombre}}`
+`sudo userdel --root {{ruta/al/otro/root}} {{nombre}}`
 
-- Elimina un usuario de un grupo:
+- Elimina un usuario junto con su directorio home y correo(mail spool):
```

Comment:
```suggestion
- Elimina un usuario junto con su directorio home y correo (mail spool):
```

Reply:
![Screenshot from 2024-10-22 21-51-25](https://github.com/user-attachments/assets/d814dc92-f48c-4ba1-8375-383e7fd8de1c)
This is how we have the current translation, just to make sure we are synced.

Reply:
`nombre` should be replaced by `nombre_usuario` as a rule of thumb, imho

Reply:
agree, I guess the intention is more clear with username. just changed it.